tests-integrations/fixtures/eslint-plugin-markdown/package.json

```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "devDependencies": {
     "eslint": "^8.0.0",
-    "eslint-plugin-markdown": "^2.0.0-0",
-    "eslint-plugin-jsonc": "file:../../../"
+    "eslint-plugin-jsonc": "file:../../../eslint-plugin-jsonc-2.3.0.tgz",
```

Comment:
You can use `resolutions` of `yarn` easilier.

```json
{
  "resolutions": {
    "eslint-plugin-jsonc": "link:../../.."
  }
}
```

Or add it in root `package.json`

```json
{
  "resolutions": {
    "eslint-plugin-jsonc": "link:."
  }
}
```

Reply:
I used `.tgz` because there were cases where it couldn't be tested correctly when the root `node_modules` was referenced.

Reply:
A root `resolutions` should have ensured this is only one `eslint-plugin-jsonc` package installed in `node_modules` as soft link.

Reply:
And also, `file:` means `copy files`, `link:` is a soft link in `yarn` as we discussed before😂

Reply:
I think that eslint referenced from eslint-plugin-jsonc will always be v8 when soft-linked. So I don't think we can test eslint v6.

Reply:
> I think that eslint referenced from eslint-plugin-jsonc will always be v8 when soft-linked. So I don't think we can test eslint v6.

Maybe, interesting, I didn't tried.

Reply:
Also, it's maybe time to drop ESLint v6 and v7 like other packages.

Reply:
I also think that support for eslint v6 needs to be dropped in the next major version. 
I'm worried if v7 support also needs to be dropped. I like to support two or more versions.

Reply:
Personally I would encourage to always use the latest versions when they are stable enough.

Reply:
> I think that eslint referenced from eslint-plugin-jsonc will always be v8 when soft-linked. So I don't think we can test eslint v6.

For testing different ESLint versions, it can be tested on CI by `yarn upgrade eslint@6` like https://github.com/prettier/eslint-plugin-prettier/blob/master/.github/workflows/ci.yml#L33

Reply:
Basic testing has already done that.
https://github.com/ota-meshi/eslint-plugin-jsonc/blob/3e2351da1ede4659c6695e25728f0d0f436adacd/.github/workflows/NodeCI.yml#L87

This test is a complex combination of modules, so I think this way is fine. We can reduce the unintended effects of devDeps.